xmipp

```diff
@@ -449,7 +449,9 @@ def compile_libsvm():
     ok = runJob("make lib")
     libDir = "src/xmipp/lib"
     os.chdir(currDir)
-    return ok and runJob("cp " + libsvmDir + "/libsvm.so.2" + " " + libDir+"/libsvm.so")
+    fnSo = os.path.join(libDir, "libsvm.so")
+    ok = ok and runJob("cp " + libsvmDir + "/libsvm.so" + " " + fnSo)
```

Comment:
`ok = ok and runJob("cp " + libsvmDir + "/libsvm.so" + " " + libDir)`
(you want to copy the file and preserve its name)